pages/index.html

```diff
@@ -4,9 +4,34 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>index</title>
+  <link href="/style/about.css" rel="stylesheet"> 
 </head>
 <body>
-  
+  <header>
+    <h1>jae clayton</h1>
+      <nav>
+          <ul>
+            <li><a href="/pages/index.html">home</a></li>
+            <li><a href="/pages/portfolio.html">portfolio</a></li>
+            <li><a href="/pages/about.html">about</a></li>
+          </ul>
+      </nav>
+  </header>
+  <main>
+    <img id="main-photo" src="/images/homepagepic.png" alt='main-photo'/>
```

Comment:
Generally, we'd want a description of the salient aspects of the image. So here, maybe something like "x-ray image of a human head".